fix critical warning for Vivado

sequencer opened the pull request:
This is mentioned in:
https://www.xilinx.com/support/documentation/sw_manuals/xilinx2018_1/ug912-vivado-properties.pdf#page=230

sequencer committed:
fix critical warning for Vivado 2020.2.

erikdanie commented:
What is the critical warning that is being fixed here?
I want to make sure we are cautious about merging stuff like this, I will want to hand run some tests.

sequencer commented:
> What is the critical warning that is being fixed here?

I’m still tuning issue here. I’ll illustrate the problem tomorrow.

sequencer committed:
disable JTAG IOB

This avoid:
[Place 30-722] Terminal 'jtag_jtag_XXX' has IOB constraint set to TRUE,
but it is either not connected to a FLOP element or the connected FLOP
element could not be brought into the I/O

[Constraints 18-841] Port jtag_jtag_XXX has IOB constraint. But it
drives multiple flops. Please specify IOB constraint on individual flop.
The IOB constraint on port will be ignored.
